config/doctrine/Alias.orm.yml

```diff
@@ -20,7 +20,7 @@ App\Entity\Alias:
       nullable: false
     destination:
       type: string
-      nullable: false
+      nullable: true
```

Comment:
We should not allow the destination to be empty, I guess postfix will use this field. We should ensure the field is filled by the user or we take the email from the user in there.

Reply:
But it should be empty for privacy reasons after the alias has been deleted.